operator/src/main/resources/instances/developer.properties

```diff
@@ -21,7 +21,7 @@ kafka.container-memory=1Gi
 kafka.jvm-xms=512M
 
 cruisecontrol.container-cpu=75m
-cruisecontrol.container-memory=256Mi
+cruisecontrol.container-memory=1Gi
```

Comment:
@k-wall @katheris since there is only a single broker will cruisecontrol even need to be enabled with developer instances?

Reply:
@shawkins that is a good point. @k-wall do you think we would ever enable cruise control for the developer instances? It feels like a production feature to me, and as Steven pointed out it can't do much in a single broker cluster anyway

Reply:
until we support multi volumes per broker, I don't think there is any value in CC for developer.